bitnami/metrics-server/values.yaml

```diff
@@ -57,6 +57,14 @@ apiService:
 ##
 securePort: 8443
 
+## Extra arguments to pass to the metrics-server
+##
+extraArgs: {}
+  # To specify extra arguments, uncomment the following lines, adjust them as necessary,
```

Comment:
I'd prefer the explanation to be above the declaration, as we do with the rest.

```yaml
## Extra arguments to pass to the metrics-server
## ref: https://github.com/kubernetes-incubator/metrics-server
##
## extraArgs:
##   kubelet-insecure-tls: true
##   kubelet-preferred-address-types: InternalIP
##
extraArgs: {}
```

What do you think about this? Similar to what it's done [in our Redis chart](https://github.com/helm/charts/blob/4802d14c93360f5373c470cd63563c3ac5a3344a/stable/redis/values.yaml#L459) and adding the link you put earlier as a reference to know the allowed arguments.

Reply:
Thank you for your time and effort on this.  I presume you noticed I'm new at this, and you have made this a positive experience.